pom.xml

```diff
@@ -95,13 +95,18 @@ under the License.
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 		<project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
 		<hadoop.version>2.8.5</hadoop.version>
+		<flink.XmxITCase>2048m</flink.XmxITCase>
+		<flink.XmxUnitTest>1024m</flink.XmxUnitTest>
 		<!-- Need to use a user property here because the surefire
 			 forkCount is not exposed as a property. With this we can set
 			 it on the "mvn" commandline in travis. -->
-		<flink.forkCount>1C</flink.forkCount>
+		<!-- Number of forkCounts for ITCase and UnitTest should take into account allocated memory
+			 to the jvm (-Xmx) and the available memory on the machine running the test -->
+		<flink.forkCountITCase>1C</flink.forkCountITCase>
+		<flink.forkCountUnitTest>2C</flink.forkCountUnitTest>
```

Comment:
I'm wondering if we shouldn't default to 2/4 respectively and remove the setting in the CI scripts. 1C/2C is a somewhat dangerous default; we already had contributors run into issue with that. (in fact I don't think my machine could handle that if the forks were running at full power).
Technically not related to this PR though 🤷

Reply:
I'm fine with that, I also wasn't sure about the 1C/2C setting in the first place. Will do it in a separate commit.